pkg/monitor.go

```diff
@@ -58,35 +62,38 @@ func New(config *Config, zapLogger *zap.Logger) *Monitor {
 			continue
 		}
 
-		err = relay.GetStatus()
+		err = relay.Status(context.TODO())
 		if err != nil {
 			logger.Warnf("relay %s has status error: %v", endpoint, err)
 			continue
 		}
 
 		relays = append(relays, relay)
-		relayMetrics[relay.ID()] = &RelayMetrics{}
+		relayMetrics[relay.NodeURL()] = &RelayMetrics{}
 	}
 
-	clock := consensus.NewClock(config.Network.GenesisTime, config.Network.SlotsPerSecond, config.Network.SlotsPerEpoch)
+	genesisTime := time.Unix(int64(config.Network.GenesisTime), 0)
+	slotsTicker := slots.NewSlotTicker(genesisTime, config.Network.SlotsPerEpoch)
+	epochTicker := helpers.NewEpochTicker(genesisTime, config.Network.SlotsPerSecond*config.Network.SlotsPerEpoch)
```

Comment:
this is likely ok but I will point out that the timing "driver" between the slots and epoch tickers are not synchronized here

they were not synchronized in the prior implementation of the `consensus.Clock` but they *were* behind a unified API where we could transparently refactor the implementation to coordinate them down the road

I think on this axis this change is a strict regression but do concede this will only be an issue around edge cases around the local clock time